src/test/java/org/fcrepo/importexport/integration/AbstractResourceIT.java

```diff
@@ -98,15 +95,22 @@ protected InputStream insertTitle(final String title) {
         }
     }
 
-    protected void assertHasTitle(final URI url, final String title) throws FcrepoOperationFailedException {
-        final FcrepoResponse getResponse = clientBuilder.build().get(url).accept("application/n-triples").perform();
-        assertEquals("GET of " + url + " failed!", SC_OK, getResponse.getStatusCode());
-        final Model model = createDefaultModel();
-        final Dataset d = new DatasetImpl(model.read(getResponse.getBody(), "", "application/n-triples"));
+    protected boolean exists(final URI uri) throws FcrepoOperationFailedException {
+        final FcrepoResponse resp = clientBuilder.build().head(uri).disableRedirects().perform();
+        return resp.getStatusCode() == 200 || resp.getStatusCode() == 307;
+    }
 
-        assertTrue(url + " should have had the dc:title, \"" + title + "\"!",
-                d.asDatasetGraph().contains(ANY, createURI(url.toString()),
-                        createProperty(DC_TITLE).asNode(), createLiteral(title)));
+    protected Model getAsModel(final URI uri) throws FcrepoOperationFailedException {
+        final FcrepoResponse getResponse = clientBuilder.build().get(uri).accept("text/turtle").perform();
```

Comment:
Cool — much tidier to use RDFDataMgr than to fetch and parse ourselves.  I've pushed a commit to do that.